privacyidea/lib/token.py

```diff
@@ -2210,8 +2210,18 @@ def check_token_list(tokenobject_list, passw, user=None, options=None, allow_res
                                                     options=options) >= 0:
                 reply_dict["serial"] = tokenobject.token.serial
                 matching_challenge = True
-                if tokenobject.is_active():
-                    # OTP matches
+                messages = []
```

Comment:
``messages`` is unused currently. I guess we should either remove it, or pass it to ``is_fit_for_challenge`` and add the messages to the reply. What do you think?

Reply:
It thinkg the messages was not filled anyways, this is why I removed it.
And I forgot to remove it from the function definition of ``is_fit_for_challenge``.

I think: Othe than dict, a list is not passed as reference and will not be filled in the function.

Reply:
So I think I should remove the messages.

Reply:
Obviously I am wrong. It would. But it looks like it does not (in our tests). I will add the messges